digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1182,13 +1205,14 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
       "elyCode" -> nodePoint.elyCode,
       "roadNumber" -> nodePoint.roadNumber,
       "roadPartNumber" -> nodePoint.roadPartNumber,
-      "track" -> nodePoint.track)
+      "track" -> nodePoint.track,
```

Comment:
is "track" -> nodePoint.track, ? or do we want "track" -> nodePoint.track.value ?

Reply:
For _javaScript_ is pretty much the same, since _Track_ has only _def value: Int_. It is being properly converter to _number_.